client/src/components/Dropdown/_Dropdown.scss

```diff
@@ -98,4 +99,26 @@
     bottom: 0px;
     margin-bottom: 4.2rem;
   }
-}
\ No newline at end of file
+
+  & .right-centered {
+    margin-left: 8.5rem;

```

Comment:
Rather than hardcoding `8.5rem`, can you use `calc(100% + 0.5rem)`? This will be the same effect, but then the margin will dynamically change if the width of the button is different (like if it has different text).